objects/rct2tt/ride/rct2.tt.cerberus.json

```diff
@@ -105,7 +105,7 @@
             "ru-RU": "Горки Цербер"
         },
         "description": {
-            "en-GB": "Sitting in comfortable trains with only simple lap restraints riders enjoy giant smooth drops and twisting track as well as plenty of 'air time' over the hills",
+            "en-GB": "Spacious trains with simple lap restraints with cars shaped like a multi-headed dog from the Underworld",
```

Comment:
```suggestion
            "en-GB": "Spacious trains with simple lap restraints with cars shaped like the multi-headed dog from the Underworld",
```

Reply:
Or if you want to be consistent with the Hyper-Twister Trains (wide cars) that the Cerberus Trains are based on, then: "Wide 4-across cars with raised seating and simple lap restraints, themed to look like the multi-headed dog from the Underworld"